### PR TITLE
[ETCM-107] Backport logging configuration

### PR DESCRIPTION
**Pull request by kapke**
# Description

Backport Logging configuration to allow:
  - logs in JSON (currently disabled, cc @mirkoAlic)
  - setting logs configuration within application.conf
  - automatic adjustment of logs placement depending on datadir setting (important for Luna)

# Important changes introduced

This PR changes logging configuration in a way that may affect deployments. How can I check whether that's true?

# Testing

1. Check whether logs are in same place as previously
2. Check whether logs follow settings under `mantis.logging`

### CODE REVIEW

File: src/main/resources/logback.xml [hunk 1]
```diff
@@ -26,8 +42,16 @@
...
+        <if condition='p("ASJSON").contains("true")'>
+            <then>
+                <appender-ref ref="STASH" />
+            </then>
+            <else>
+                <appender-ref ref="STDOUT" />
+            </else>
+        </if>
         <appender-ref ref="FILE" />
+        <appender-ref ref="METRICS" />
```
Comment (mirkoAlic) [inline src/main/resources/logback.xml:54]: should be included now?

**Review by mirkoAlic — APPROVED** [review]
(only one minor comment/doubt) LGTM!

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -536,6 +536,17 @@ mantis {
     # Iff true, any errors during metrics client operations will be logged.
     log-errors = true
   }
+
+  logging {
```
Comment (mmrozek) [inline src/main/resources/application.conf:540]: Shouldn't we expose this config in the universal folder also? Maybe the file `logging.conf` will be useful.

File: src/main/resources/logback.xml (hunk 1 above)
Comment (kapke) [reply src/main/resources/logback.xml:54]: Not! I did remove appender but forgot to remove ref to it

**Review by mmrozek — APPROVED** [review]
LGTM

**Pull request merged**